DEV/org.openl.core/src/org/openl/types/java/JavaImportTypeLibrary.java

```diff
@@ -46,6 +48,19 @@ public JavaImportTypeLibrary(String[] importPackages, String[] importClasses, Cl
                 }
             }
         }
+
+        // For backward compatibility with old types in rules
+        aliases.put("BigDecimalValue", JavaOpenClass.getOpenClass(BigDecimal.class));
+        aliases.put("DoubleValue", JavaOpenClass.getOpenClass(Double.class));
+        aliases.put("DoubleValuePercent", JavaOpenClass.getOpenClass(Double.class));
+        aliases.put("FloatValue", JavaOpenClass.getOpenClass(Float.class));
+        aliases.put("BigIntegerValue", JavaOpenClass.getOpenClass(BigInteger.class));
+        aliases.put("LongValue", JavaOpenClass.getOpenClass(Long.class));
+        aliases.put("IntValue", JavaOpenClass.getOpenClass(Integer.class));
+        aliases.put("ShortValue", JavaOpenClass.getOpenClass(Short.class));
+        aliases.put("ByteValue", JavaOpenClass.getOpenClass(Byte.class));
+        aliases.put("StringValue", JavaOpenClass.getOpenClass(String.class));
+        aliases.put("ObjectValue", JavaOpenClass.getOpenClass(Object.class));
     }

```

Comment:
Backward compatibility for used full class name is also required.

Reply:
added